{{cookiecutter.github_repository}}/Makefile

```diff
@@ -6,7 +6,7 @@ PROJECT_NAME={{cookiecutter.main_module}}
 DB_NAME=$(PROJECT_NAME)
 INVENTORY=provisioner/hosts
 PLAYBOOK=provisioner/site.yml
-ENV_PREFIX=$(shell python -c "if __import__('pathlib').Path('.venv/bin/pip').exists(): print('.venv/bin/')")
+ENV_PREFIX=$(shell python -c "if __import__('pathlib').Path('venv/bin/pip').exists(): print('venv/bin/')")
```

Comment:
A better way to go about it is doing some checks for the existence of virtual env and setting the `ENV_PREFIX` to either a path or `poetry run ` and this will also avoid typing `poetry run ` in every command.

Also, poetry creates .venv by default so it's probably better to start adopting `.venv` as virtualenv dir instead of `venv`.

pseudo code
```
if exists ".venv/bin/pip"
   ENV_PREFIX=".venv/bin/"
if exists "venv/bin/"
   ENV_PREFIX="venv/bin/"
if $(which poetry)
  ENV_PREFIX="poetry run "
else 
  ENV_PREFIX=""
```

Reply:
I ended up with `poetry run` here. 

Poetry unless specified to have venvs inside the project repo keeps it in cached directory outside of all the projects by default.

I've also removed direct venv references elsewhere.